add-on/panels/js/roomToc.jsx

```diff
@@ -12,51 +13,80 @@ loop.roomToc = (function() {
   function init() {
     loop.shared.utils.getBoolPreference = function foo() {};
 
-    var dispatcher = new loop.Dispatcher();
-    var sdkDriver = new loop.OTSdkDriver({
-      // For the standalone, always request data channels. If they aren't
-      // implemented on the client, there won't be a similar message to us, and
-      // we won't display the UI.
-      constants: {},
-      useDataChannels: true,
-      dispatcher: dispatcher,
-      sdk: OT
-    });
+    var requests = [
+      ["GetAllConstants"],
+      ["GetAllStrings"],
+      ["GetLocale"],
+      ["GetLoopPref", "ot.guid"]
+    ];
 
-    var activeRoomStore = new loop.store.ActiveRoomStore(dispatcher, {
-      sdkDriver: sdkDriver
-    });
+    loop.requestMulti.apply(null, requests).then(function(results) {
+      // `requestIdx` is keyed off the order of the `requests` and `prefetch`
+      // arrays. Be careful to update both when making changes.
+      var requestIdx = 0;
+      var constants = results[requestIdx];
 
-    // XXX akita: Get loop constants
-    var roomStore = new loop.store.RoomStore(dispatcher, {
-      constants: { CONSTANT: false }
-    });
+      // Do the initial L10n setup, we do this before anything
+      // else to ensure the L10n environment is setup correctly.
+      var stringBundle = results[++requestIdx];
+      var locale = results[++requestIdx];
```

Comment:
We can use destructuring assignment and just do `let [constants, stringBundle, locale] = results`